src/core/consts.c

```diff
@@ -39,9 +41,12 @@ ROX_INTERNAL size_t _rox_env_return_value_using_mode_env(
             return str_copy_value_to_buffer(buffer, buffer_size, qa_mode_value);
         } else if (str_equals(value, ROX_ENV_MODE_LOCAL)) {
             return str_copy_value_to_buffer(buffer, buffer_size, local_mode_value);
-        }
+        } else if (str_equals(value, ROX_ENV_MODE_PRODUCTION)) {
+            return str_copy_value_to_buffer(buffer, buffer_size, prod_mode_value);
+        } 
     }
-    return str_copy_value_to_buffer(buffer, buffer_size, prod_mode_value);
+    // Default to the platform
+    return str_copy_value_to_buffer(buffer, buffer_size, platform_mode_value);
 }
```

Comment:
we went away from env var
other SDKs have in RoxOptions a configuration or network configuration property, which includes all URLs
guess this SDK was never updated, this will mean no support for VPC, and can't run it in PLATFORM preprod..
probably need to ask Glen if it should be added.. or just not support it

Reply:
Could we make the environment variable a new PR?  This one is pretty large already, and I'd rather do it in a new set of work.  Another positive is that's it's like a new release when we merge so it's not a large lift in terms of getting a new feature out there.

Reply:
if the default is back at Rollout-production we can continue other things in a different PR
we can't set the default to be PLATFORM right now IMO

i'm also not sure how versions will work here, i will imagine that it just the latest one, or need to specify a commit, so better change the version as part of this PR as well

Reply:
I'll switch the default to be rollout-production which would negate the changes to a lot of the tests.

There aren't versions for this SDK unfortunately; our install directions are to basically clone the directory, run the install script for the SDK, and to go forward from there.  It doesn't seem like the best user experience, but I don't have any experience with developing server-side products in c.

Reply:
think we need to discuss with @glen-polo or @elcloudbees about the ROI for supporting this SDK on CBP right now (before anybody is using it)